api/src/main/java/jakarta/servlet/http/LocalStrings.properties

```diff
@@ -19,8 +19,10 @@
 # Default localized string information
 # Localized for Locale en_US
 
-err.cookie_name_is_token=Cookie name \"{0}\" is a reserved token
+err.cookie_name_is_token=Cookie name \"{0}\" contains a reserved token
 err.cookie_name_blank=Cookie name must not be null or empty
+err.cookie_attribute_name_is_token=Cookie attribute name \"{0}\" contains a reserved token
```

Comment:
I think this should be:
`Cookie attribute name \"{0}\" contains an invalid character for an attribute name.